agent-control/src/http/client.rs

```diff
@@ -1,6 +1,7 @@
 //! # Helpers to build a reqwest blocking client and handle responses and handle responses
 //!
 use crate::http::config::HttpConfig;
+use crate::utils::certificates::{cert_paths_from_dir, certificate_error};
```

Comment:
Another alternative would be keeping the helpers here and make them public (after all the certificates are for http client even if such client is used by the oci_client under the hood). 

I usually try to avoid `utils` packages but that's probably just me.